### PR TITLE
[feature] Added option to specify host url #102

### LINKED ISSUE #102: [wifi-login-app] Add openwisp-radius URL parameter to add-org configuration command
The add-org configuration generator is missing a very important parameter, the openwisp-radius URL is currently not asked to the user. We should ask for this information, making it clear that it will default to `http://127.0.0.1:8000` if left empty.

The new doc section added in #101 needs to be updated to mention this new option.

### PR DESCRIPTION
**Pull request by NoumbissiValere**
Closes #102

### CODE REVIEW

File: client/components/mobile-phone-verification/mobile-phone-verification.js [hunk 1]
```diff
@@ -141,51 +186,6 @@ export default class MobilePhoneVerification extends React.Component {
...
-        });
-      });
-  }
-
-  async handleLogout() {
-    const {orgSlug, logout, cookies} = this.props;
-    logout(cookies, orgSlug);
-    toast.success(logoutSuccess);
-  };
-
```
Comment (nemesisdesign) [inline client/components/mobile-phone-verification/mobile-phone-verification.js:188]: why are you changing this?

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Looks good, will test it soon. Please rebase on the current master, avoid changing other parts of the code in this PR.

File: internals/generators/organization/index.js [hunk 2]
```diff
@@ -77,6 +77,12 @@ module.exports = {
         return "The captive portal logout action url required";
       },
     },
+    {
+      type: "input",
+      name: "host",
```
Comment (nemesisdesign) [inline internals/generators/organization/index.js:82]: I would call it openwisp_radius_url rather than `host`, to make it more obvious.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: client/components/mobile-phone-verification/mobile-phone-verification.js (hunk 1 above)
Comment (nemesisdesign) [reply client/components/mobile-phone-verification/mobile-phone-verification.js:188]: I see, it shouldn't happen on the latest master, if it does, let me know and I'll fix master (but the build is passing so it should not happen).

Comment (NoumbissiValere) [reply client/components/mobile-phone-verification/mobile-phone-verification.js:188]: ok

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**